### DISCUSSION TITLE
Creating a Canvas with a backgroundImage

### DISCUSSION THREAD

**Discussion by johnthad** [fabricjs/fabric.js#8902] [Q&A] [answered]
I'm attempting to assign an image as a Canvas's `backgroundImage` using v6.0.0-beta5. The image is read--the canvas's dimensions update to reflect the image size--but the image is not displayed. This is true regardless of `backgroundImage` being set to the image element or the image's URL (which is correct?).

See: https://codesandbox.io/s/fabric-6-background-nbzrn6

What am I doing wrong, or is `backgroundImage` not yet supported in fabric 6.0.0-beta?

**Comment by ShaMan123** [comment, marked as answer]
https://codesandbox.io/s/fabric-6-background-forked-7gjhkp?file=/src/index.js

**Reply by ShaMan123** [reply]
you need to create a fabric.Image and render the canvas

**Reply by johnthad** [reply]
Thank you so much. I'm looking forward digging into Fabric.js.
